operator/v1/types_console.go

```diff
@@ -21,13 +21,27 @@ type ConsoleSpec struct {
 	// customization is used to optionally provide a small set of
 	// customization options to the web console.
 	// +optional
-	Customization ConsoleCustomization `json:"customization"`
+	Customization ConsoleCustomization `json:"customization,omitempty"`
+	// providers contains configuration for using specific service providers.
+	Providers ConsoleProviders `json:"providers,omitempty"`
 }
 
 type ConsoleStatus struct {
 	OperatorStatus `json:",inline"`
 }
 
+type ConsoleProviders struct {
+	// statuspage contains ID for statuspage.io page that provides status info about.
+	// +optional
+	Statuspage *StatuspageProvider `json:"statuspage,omitempty"`
+}
+
+// StatuspageProvider provides identity for statuspage account.
+type StatuspageProvider struct {
+	// pageID is the statuspage ID.
```

Comment:
This doesn't explain in details what that is, honestly. The current description is just saying "page id is statuspage id", please explain a bit more.

Reply:
I'd suggest:

"The unique ID assigned by Statuspage for your page. This must be a public page."